cli/jobs/dataprep/dask/readme.md

```diff
@@ -0,0 +1,48 @@
+---
+page_type: sample
+languages:
+- azurecli
+- python
+products:
+- azure-machine-learning
+description: This sample shows how to run a distributed DASK job on AzureML. The 24GB NYC Taxi dataset is read in CSV format by a 4 node DASK cluster, processed and then written as job output in parquet format. 
+---
+
+Running a DASK job
+------------------
+
+This example shows how a distribted DASK job can be run on multiple nodes of a cluster. In this example we are using 4 nodes 
+using this job yaml. The startup of the cluster is done by the `startDask.py` script which launches a scheduler
+and a worker on the first node of the cluster and a worker on all the other nodes.
+
+
+>  Since the script is writing the output to the local drive, and since the dataset is cached on the same local drive,
+  your cluster nodes need to have enough free space on the local volume to accomodate pretty much the whole input and output datasets used (to be on the safe side). 
+  The input dataset will be 24GB, the parquet output is about 4GB. Using STANDARD_D15_V2 VMs to build your cluster will give you close to 1TB of
+  free disk space and works well even for bigger datasets.
+
+If a --script parameter is provided, then the script will run that script after the cluster has been brought up and the job 
+will be terminated after the script has completed. To start a DASK cluster for interactive work, don't provide a --script parameter, 
+which will have the job run indefinitely (i.e. until you terminate it).
+
+The job below is currently launched with `distribution: .. type: pytorch` since that gives the full flexibility of assigning the work to the 
+different nodes of the cluster by just checking the $RANK environment variable. In the future we will provide a more generic name for that mode of launching a distributed job.
+
+For debugging and interactive work, the script also launches a Jupyter server on the first node which can be accessed by ssh tunnelling into
```

Comment:
do we want to include this logic in the script, and inform users? seems to add a lot of complication